readme.md

```diff
@@ -287,6 +287,65 @@ See the [AWS section](#aws) for an example.
 
 **Note**: Modifying the `body` is not recommended because the `content-length` header has already been computed and assigned.
 
+###### events
+
+Type: `Object<string, Function[]>`<br>
+Default: `{}`
+
+Events allow listeners to be attached to all events emitted during the request lifecycle. Listeners are subscribed using `on` or `once` according to the known frequency of the event.
```

Comment:
Can you place everything below here in a `<details>` tag so it won't take up so much space in the readme?